blog/_posts/2022-10-02-compiler-academy.md

```diff
@@ -0,0 +1,37 @@
+
+A lot changed over the past year. We've successfully conducted over 20 iterations of the Spree, gradually opened it to motivated learners from outside our core teams, polished our workflows, and started a [YouTube channel](https://www.youtube.com/channel/UCIH0OgqE54-KEvYDg4LRhKQ) where we regularly post educational videos about how the compiler works.
+
+The Compiler Academy started to gain more and more presence in the community, and more and more people became interested in joining. Today, we are taking a logical step towards growing this community of compiler enthusiasts even further. We're moving all the organization and communication around the Compiler Academy to the [Scala Discord server](https://discord.com/invite/scala). Now, you can follow the spree at the #scala-contributors channel of that Discord, where all the spree announcements will be regularly posted.
+
+If you're a lifelong learner passionate about all things programming, you may have wondered what happens when your favorite language compiles your code. Maybe you have aspirations to contribute your ideas and solutions to the compiler one day. Or maybe you're eager to understand how a modern cutting-edge compiler works because you'd like to develop a programming language of your own.
+
+Scala 3 Compiler Academy is your chance to become a part of the community of common-minded people and learn the compiler together!
+
+## Rules of Participation
+The Spree is organized by the Scala Center. We are committed to keeping the Spree community professional and friendly, so before you can join any of the Sprees, you will need to complete the one-time application process including an interview with a Scala Center representative. While participating in the official Scala spaces, including the official Discord server and the Compiler Academy community, you are required to follow the [Code of Conduct](https://www.scala-lang.org/conduct/). Scala Center reserves the right to refuse your participation in the Compiler Academy at any point at our sole discretion.
+
+## Logistics
+Scala 3 Compiler Academy Issue Spree is an online event that takes part every 3 weeks – on Tuesdays at 17:00-19:00 CET. Before participating in the sprees, you will meet with a Scala Center representative for a short onboarding interview. After that, you will be able to register for each event separately, possibly specifying your preferences about the issue you'll be working on. You then will be assigned to a team of 2-3 people, and during the spree, you'll be working on the same issue (not necessarily the one you specified in your registration) in a Zoom breakout room.
```

Comment:
```suggestion
Scala 3 Compiler Academy Issue Spree is an online event that takes part every 3 weeks – on Tuesdays at 17:00-19:00 CE(S)T. Before participating in the sprees, you will meet with a Scala Center representative for a short onboarding interview. After that, you will be able to register for each event separately, possibly specifying your preferences about the issue you'll be working on. You then will be assigned to a team of 2-3 people, and during the spree, you'll be working on the same issue (not necessarily the one you specified in your registration) in a Zoom breakout room.
```
Depending on the time of year, it can be CEST (although we just changed into CET last weekend, so it will be CET for the next 6 months).